.github/workflows/k8s-content-pr-test.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: Gate / Kubernetes Test Content Parsing
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  datastream-parsing:
+    name: XCCDF Datastream parsing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Copy XCCDF files from existing content image
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 15
+          max_attempts: 3
+          retry_wait_seconds: 120
+          retry_on: error
+          command: |
+            mkdir -p content
+            docker pull ghcr.io/complianceascode/k8scontent:${{ github.event.number }}
+            docker run --rm -v $PWD/content:/content:z ghcr.io/complianceascode/k8scontent:${{ github.event.number }} bash -c "cp *.xml /content"
+      - name: Clone compliance operator repository
+        run: |
+          git clone https://github.com/ComplianceAsCode/compliance-operator.git
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '>=1.19.0'
```

Comment:
Are we going to have to bump this go version separately each time we bump it in the compliance-operator?

Reply:
I think it will use the latest go, 
```
Run go version
go version go1.22.0 linux/amd6[4](https://github.com/ComplianceAsCode/content/actions/runs/8007294791/job/21871061364?pr=11615#step:5:5)
```

Reply:
Oh - this is just enforcing a lower bound. Thanks.

Reply:
We will be testing with a version of golang (1.22) that's potentially different from the version of golang we build the operator with, correct?

So there would be a chance that the tests we invoke using https://github.com/ComplianceAsCode/compliance-operator/pull/493/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R589 would break against the latest version of golang we're using here?

Reply:
that's a good point, I changed this part to fetching the go version from the CO go.mod file